x-pack/plugins/event_log/generated/mappings.json

```diff
@@ -309,19 +309,6 @@
                         }
                     }
                 },
-                "reporting": {
```

Comment:
Normally, mappings can never be removed, but in this case this mapping was added and being removed in  8.1 PRs.